src/etl/gwas_ingest/process_associations.py

```diff
@@ -37,6 +37,47 @@
 }
 
 
+def filter_assoc_by_maf(df: DataFrame) -> DataFrame:
+    """
+    > We take the maximum minor allele frequency across all populations, and then drop all but the row
+    with the highest minor allele frequency from all mappings. This process is population name agnostic,
+    the only assumption is the field name is `allele_frequencies` and it is a structType.
+
+    Args:
+      df (DataFrame): DataFrame
+
+    Returns:
+      A DataFrame with identical columns
+    """
+    # parsing population names from schema:
```

Comment:
Not clear to me how this info is encoded. Are the populations inside an array of structs?

Reply:
Yeah, I shouldn't have relied on the automatic document generation. I'll clarify.